README.md

```diff
@@ -134,6 +134,30 @@ module.exports = {
   }
 }
 ```
+
+### fonts
+
+You can add custom fonts to the config viewer that are used
+in your Tailwind config
+```js
+module.exports = {
+  theme: {
+    // ...your Tailwind theme config
+    configViewer: {
+      fonts: [
+        {
+          fontFamily: 'Open Sans',
+          fontWeight: '400',
+          fontStyle: 'normal',
+          fontDisplay: 'swap',
+          src: "url(https://fonts.googleapis.com/css2?family=Open+Sans&display=swap)"
```

Comment:
@acharlop Instead of defining this font object, could we simply specify the css file URL only?

```
fontCssUrls: [
  'https://fonts.googleapis.com/css2?family=Open+Sans&display=swap'
]
```

Then load that CSS file in the viewer like you are doing now. Perhaps I'm not understanding why we would need the other properties in this object.

Reply:
You're right that it should handle an url string in the array also. I think it should also handle full objects so that if you already have font face css defined you can reuse the entire definition. Usually for different font weights, perhaps combining font families and font weights section could be useful?

Reply:
Good question about combing the family and weight sections. However, I'm inclined to keep them separate since Tailwind does as well and I'm sure they made that decision for a good reason.

If we only require a URL to a CSS file that contains the @font-face rulesets, you can remove all of the code you have for generating the config.css file and serving from the Koa server. I think this simplifies it quite a bit.

Locally I tested it out with only loading the font CSS files and it worked well:

https://monosnap.com/file/RyyDQHkFRM1AY1DaWbRsLlWnPbEckE
https://monosnap.com/file/305WZBV92b3difmrgD2YvTvPRuWwlC

One thing to note is if the CSS file defines multiple weights for the font, the Font Family section defaults to the last weight defined in the CSS file. For Roboto in the screenshots above, it is displaying at 500 weight instead of 100 since 500 is defined after 100 in the CSS file https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap

In this situation, I think your suggestion of accepting either a string to a CSS file URL that contains the @font-face rulesets or an object so you can at least set the weight you want the font to display at if you have multiple weights could be helpful. We can separate the css file definitions from the weights in the config:

```
fonts: {
  // string
  // css: 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap',
  // or an array of strings
  css: [
    'https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap',
    'https://fonts.googleapis.com/css2?family=Open+Sans&display=swap'
  ],
  // (Optional) Set the weight you want the font to display at in the font family section
  displayWeights: {
     // the key should match the name you used for the font in the fontFamilies section of the tailwind config
     roboto: 100  
  }
}
```

Then in the font family section, you can check if a display weight is defined for the font you are outputting and inline it if it is set.

Reply:
In that case I think just noting that a single font weight should be put in each url and if multiple weights are in there only the last will be used is much simpler

Reply:
The reason I still think it should be an object though is that we need some way to map `fontFamily` to a url.

```
theme: {
  fontFamily: {
    primary: ['Roboto'],
    secondary: ['Open Sans']
  },
  // ... 
  configViewer: {
    fonts: {
      Roboto: "https://fonts.googleapis.com/css2?family=Roboto&display=swap"
      'Open Sans': "https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
    }
  }
}
```

Reply:
Disregard my last message. Realized an array is fine.